app/models/broadcast_message.py

```diff
@@ -298,10 +322,26 @@ def _update_areas(self, force_override=False):
 
         self._update(**data)
 
+    def _update_custom_areas(self, force_override=False):
+        areas = {
+            "ids": self.area_ids,
+            "names": [area.name for area in self.areas],
+            "aggregate_names": [area.name for area in aggregate_areas(self.areas)],
+            "simple_polygons": self.simple_polygons.as_coordinate_pairs_lat_long,
+        }
+
+        data = {"areas": areas}
+
+        # TEMPORARY: while we migrate to a new format for "areas"
+        if force_override:
+            data["force_override"] = True
```

Comment:
Possibly create a ticket for this and link to it in the comment, so that anyone who wants to remove it can see if it's been done or not.

Reply:
Ticked created and linked in comment.